react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import Icon from '../../icons/Icon';
+
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText
+}) => (
+  <button className="ma__button-alert" onClick={onClick}>
+    <span className="ma__button-alert__show">{showText}</span>
+    <span className="ma__button-alert__hide">{hideText}</span>
+    <Icon />
+    {text}
+  </button>
+);
+
+ButtonAlert.propTypes = {
```

Comment:
When viewing a story, if you click on the `Show Info` button near the bottom right, you'll get to see extra information about the component. Included is this is a description of each prop, which actually comes from this file when there's comments above each prop like this:
```
/** This is an example description. */
```

You do need the format to be exactly `/** */`. Go ahead and add a small description of what each prop does for this component.